packages/modules/extension-ads/src/content/content.js

```diff
@@ -686,8 +686,12 @@ if ( typeof vAPI === 'object' && !vAPI.contentScript ) {
           }
       }
       if (node.localName === "iframe" && node.contentWindow && node.ownerDocument === document && node.getAttribute("role") !== "presentation") {
-        this.highlightNodeAsAds(node, _indent, "red", "urlIsAnAd-hit", node.src);
-      }
+        const iframeSrc = node.src;
+
+        if (!iframeSrc.includes("youtube") && !iframeSrc.includes("google.com")) {
```

Comment:
Can we move it to the filter config?

Reply:
No, if we do this, then all blocks with these src will be marked as ads